setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='edx-opaque-keys',
-    version='0.3.2',  # Please update CHANGELOG when you bump versions
+    version='0.3.1',
```

Comment:
If I bump this right now, this very commit will get uploaded to pypi as 0.3.3. Is that what we want?